plugin/pkg/scheduler/api/validation/validation.go

```diff
@@ -34,5 +34,10 @@ func ValidatePolicy(policy schedulerapi.Policy) error {
 		}
 	}
 
+	for _, extender := range policy.ExtenderConfigs {
```

Comment:
not positive on the context, but presumably a nil TLSConfig means use the defaults (system roots, hostname verification on)